src/AutoRest.CSharp/Common/Input/InputTypes/InputOperation.cs

```diff
@@ -30,7 +30,9 @@ internal record InputOperation(
     OperationPaging? Paging,
     bool GenerateProtocolMethod,
     bool GenerateConvenienceMethod,
-    bool KeepClientDefaultValue)
+    bool KeepClientDefaultValue,
+    string? OperationId,
```

Comment:
we don't need Operationid, we can change the description

Reply:
would like to keep `OperationId` for swagger input, because the user is using this to locate the corresponding operation in swagger.